src/mutable_data.rs

```diff
@@ -689,6 +685,12 @@ pub struct SeqEntryActions {
 }
 
 impl SeqEntryActions {
+    /// Create a new Unsequenced Entry Actions list
+    pub fn new() -> Self {
+        let actions: BTreeMap<Vec<u8>, SeqEntryAction> = Default::default();
```

Comment:
Alternate implementation: just derive `Default` for `SeqEntryActions` (since `Btreemap` implements`Default`) and then this function can just be:

```rust
    pub fn new() -> Self {
        Default::default()
    }
```

This would save a few lines :) Same for `UnseqEntryActions` below.